api/managementpb/backup/common.proto

```diff
@@ -18,3 +20,25 @@ enum BackupMode {
   INCREMENTAL = 2;
   PITR = 3;
 }
+
+// File represents file or folder on a storage.
+message File {
+  string name = 1;
+  bool is_directory = 2;
+}
+
+// ReprBackup contains additional data for backup tools.
+message ReprBackup {
+  // Name of backup in backup tool representation.
+  string name = 1;
+}
+
+// Repr represents artifact on a storage.
+message Repr {
```

Comment:
That name is not semantic. I see we are going to use it a lot in the future, so let's see if we can come up with a better one.

I suggest we consider those two as alternative names:
- `ArtifactStorageRec`
- `StorageRec`

Reply:
Changed naming.